src/chainlit/message.py

```diff
@@ -213,6 +213,8 @@ async def send(self):
         trace_event("send_message")
         id = await super().send()
 
+        self.emitter.session.root_message = self
```

Comment:
should we check i that message has not parent id to set it as the root message?

Reply:
good catch, thanks!